net-libs/msgraph/msgraph-0.2.3.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org meson
+
+DESCRIPTION="GLib-based library for accessing online serive APIs using MS Graph protocol."
+HOMEPAGE="https://gitlab.gnome.org/GNOME/msgraph"
+
+LICENSE="LGPL-2+"
+SLOT="0/1"
+KEYWORDS="~amd64 ~arm ~arm64 ~loong ~ppc64 ~riscv ~sparc ~x86"
+
+IUSE="debug gtk-doc +introspection man"
+
+RDEPEND="
+	>=dev-libs/glib-2.28.0:2
+	dev-libs/json-glib
+	net-libs/rest
+	>=net-libs/libsoup-3.0:3.0
```

Comment:
Any reason for not simply using net-libs/libsoup:3.0? Looking to the meson file, it seems that also their "alpha" versions would work as they will also provide 'libsoup-3.0'
https://gitlab.gnome.org/GNOME/msgraph/-/blob/main/meson.build?ref_type=heads#L30